tests/antithesis/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
         chown -R ${USER_ID:-root}:${GROUP_ID:-root} /var/etcddata0 /var/etcddata1 /var/etcddata2 /var/report
 
   etcd0:
-    image: 'etcd-server:latest'
+    image: 'etcd-server:${REF:-latest}'
```

Comment:
This will not work because the Antithesis doesn't template the file. We would also need to render this templating into a config file.

Reply:
Sorry about the confusion.  I went to read Antithesis's documentation and now understand what you mean.  This newly pushed one should do what is required... I think.